package.json

```diff
@@ -43,8 +36,7 @@
     "@types/react-dom": "^18",
     "@typescript-eslint/eslint-plugin": "^5.59.2",
     "@typescript-eslint/parser": "^5.59.2",
-    "babel-jest": "^27.2.5",
-    "jest-transform-stub": "^2.0.0",
```

Comment:
Yarn moved some of these dependencies around for some reason, but they should all have still have the same version, so I left it as-is.